ha-singleton-service/README.md

```diff
@@ -168,58 +220,96 @@ while all other nodes will log:
+
+        /subsystem=singleton/singleton-policy=default/election-policy=simple:remove(){allow-resource-service-restart=true}
+        /subsystem=singleton/singleton-policy=default/election-policy=random:add()
+
+3. Open a new command prompt, navigate to the root directory of this quickstart, and run the following command to execute the script for Server 1. Be sure to replace ${jboss.home.name}-1 with the path to the target Server 1.
+
+        For Linux: ${jboss.home.name}-1/bin/jboss-cli.sh --connect --file=random-election-policy-add.cli
+        For Windows: ${jboss.home.name}-1\bin\jboss-cli.bat --connect --file=random-election-policy-add.cli
+    You should see the following result when you run the script.
+
+        The batch executed successfully
+        process-state: reload-required
+    Note that the `random-election-policy-add.cli` batch script executes the `reload` command, so a reload is not required.
+
+4. Stop the server and review the changes made to the `standalone-ha.xml` server configuration file by the management CLI commands. The `singleton` subsystem now contains a  `random-election-policy` element under the `singleton-policy` that specifies the preferences `node3 node2 node1`.
+
+        <subsystem xmlns="urn:jboss:domain:singleton:1.0">
+            <singleton-policies default="default">
+                <singleton-policy name="default" cache-container="server">
+                    <random-election-policy/>
+                </singleton-policy>
+            </singleton-policies>
+        </subsystem>
+
+  5.  Repeat these steps for the second server. Note that if the second server is using a port offset, you must specify the controller address on the command line by adding `--controller=127.0.0.1:10090`.
+
+        For Linux: ${jboss.home.name}-2/bin/jboss-cli.sh --connect --controller=127.0.0.1:10090 --file=random-election-policy-add.cli
+        For Windows: ${jboss.home.name}-2\bin\jboss-cli.bat --connect --controller=127.0.0.1:10090 --file=random-election-policy-add.cli
+
+6. Be sure both servers are started, deploy one of the examples to both servers, and verify that the election policy is now in effect.
+
+### Configure a Quorum Singleton Policy
+
+A quorum policy specifies the minimum number of cluster members that must be present for the election to even begin. This mechanism is used to mitigate a split brain problem by sacrificing the availability of the singleton service. If there are less members than the specified quorum, no election is performed and the singleton service is not run on any node.
+
+1. Start the two servers with the HA profile as described above.
+2. Review the contents of the `quorum-add.cli` file located in the root of this quickstart directory. This script specifies the minimum number of cluster members required for the singleton policy using this command.
```

Comment:
"This script specifies the minimum number of cluster members required for the singleton policy using this command."
Can we ditch this sentence? Mostly its cumbersome, but what quorum actually is, is explain above and this is not precise.